src/button/types.ts

```diff
@@ -2,13 +2,13 @@ import React from "react";
 import { ComponentLoadingSpinnerProps } from "src/shared/component-loading-spinner/component-loading-spinner";
 
 type StyleType = "default" | "secondary" | "light" | "link";
-
 export interface ButtonProps
     extends React.ButtonHTMLAttributes<HTMLButtonElement> {
     /** The style type of the button. Values: "default" | "secondary" | "light" | "link" */
     styleType?: StyleType | undefined;
     /** Indicates if a loading spinner is to be displayed */
     loading?: boolean | undefined;
+    danger?: boolean | undefined;
```

Comment:
Do you have any mockups that I can refer to? The reason this is not a variant is because there is the concept of light and secondary but in red is it?